public/client.js

```diff
@@ -2,9 +2,26 @@ $(function () {
 
 	'use strict';
 
+    /*
+     * QUESTION:  I wanted to add underscore here in order to use
+     * _.sortBy to display trucks in alpha order, but I got the 
+     * RefError 'require is not defined'.  What am I doing wrong?
+     *
+     * var _ = require('underscore');    
+     */
+    
 	var foodTypes = [];
 
 	$.get('/trucks', function (truckList) {
+
+        /*
+         * This is where I wanted to sort but couldn't get
```

Comment:
Just add a script reference to index.html that includes the underscore library:

`<script src="https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js"></script>`

Make sure it appears before the client.js script reference.